test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Link.html

```diff
@@ -4,10 +4,75 @@
 <head>
     <meta charset="utf-8" />
     <title>Link</title>
-    <link href="/blank.css" rel="stylesheet"></link>
-    <link rel="stylesheet" href="/link-test.min.css" data-extra="test" />
-<meta name="x-stylesheet-fallback-test" class="hidden" />
-<script>!function(a,b,c){var d=document,e=d.getElementsByTagName("SCRIPT"),f=e[e.length-1].previousElementSibling;d.defaultView.getComputedStyle(f)[a]!==b&&d.write('<link rel="stylesheet" href="'+c+'"/>')}("visibility","hidden","/link-test.css");</script>
+
+    <!-- Plain link tag -->
+    <link href="/site.css" rel="stylesheet"></link>
```

Comment:
Confused, only difference between this link tag and the one on line 12 is self/closing vs not right? What's up with the "Globbed link tag with existing file" comment below?

One thing to note: Link tags aren't allowed to have end tags according to the HTML5 spec; i don't mind showcasing that they can be used this way with tag helpers though.